### ISSUE TITLE
Support for "https://openconceptlab.org" (omitting "www")

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/DATIM-Metadata-Project#121]
Is it possible to have openconceptlab.org also point to www.openconceptlab.org?

This is how the Digital Ocean servers were set up, and I know there are lots of links in documents in Ethiopia that have that URL in it :)

Not super high priority though...

**Comment by rkorytkowski** [comment]
Initially I configured a redirect from openconceptlab.org to www.openconceptlab.org and it worked in Firefox on Windows, Chrome on Android, but did not work in Chrome on Windows... I assume you also tested with Chrome... I couldn't determine why it was failing (it was improperly handled by godaddy) so I changed openconceptlab.org to point to IP of nairobi.openmrs.org. The issue is that when we need to move the server to a different machine then the IP will change and we will have to adjust DNS entry for openconceptlab.org (unlike other entries which point to *.openmrs.org). Typically IP should not change unless we are forced to re-provision the machine due to a vm failure (should not happen in normal conditions).
